### DISCUSSION TITLE
Access to my authorizations google account toz account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#137357] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

Access to my authorizations google account toz account  
Remark mz android device was hacked from  unknown device about my private things

**Comment by queenofcorgis** [comment]
Hi @Kasalongbymak ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Google like their [Account Help Community](https://support.google.com/accounts/community?hl=en). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
